packages/opni/opni/package.yaml

```diff
@@ -1,6 +1,6 @@
 
 url: local
-version: 0.5.4
+version: 0.5.5-dan-dev
```

Comment:
Intentional change?

Reply:
Yes, but probably shouldn't be committed to main